packages/react-reconciler/src/ReactFiberCompleteWork.js

```diff
@@ -739,7 +737,7 @@ function completeDehydratedSuspenseBoundary(
   ) {
     warnIfUnhydratedTailNodes(workInProgress);
     resetHydrationState();
-    workInProgress.flags |= ForceClientRender | Incomplete | ShouldCapture;
+    workInProgress.flags |= ForceClientRender | DidCapture;
```

Comment:
I believe this was required because of the dev warning about incomplete fibers on the completwork path but I'm trying to wrap my head around the semantics of ShouldCapture -> DidCapture. Like I get that the ForceClientRender flag is what now triggers the client fallback render (which sorta seems like what it always should have been) but does marking DidCapture change any of the work loop behaviors? I haven't connected it to the rest of the changes in my mental model

Reply:
It should have been `DidCapture` all along, it just happened to work the other way, too, though it was doing some redundant work.

I can't actually remember the reason they were ever separate but now that we always immediately unwind to the boundary after something throws, we might not need either of them...

Reply:
I extracted this part to its own PR with a test demonstrating the change: https://github.com/facebook/react/pull/26445